ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/CreateVulnReport/CreateVulnReportPage.tsx

```diff
@@ -37,8 +38,21 @@ function VulnReportsPage() {
                     </FlexItem>
                 </Flex>
             </PageSection>
-            <PageSection padding={{ default: 'noPadding' }}>
-                <PageSection isCenterAligned />
+            <Divider component="div" />
+            <PageSection padding={{ default: 'noPadding' }} isCenterAligned>
```

Comment:
nit: Is the `padding` prop necessary?

Reply:
Yea, there are two reasons for that. 

One is that it won't look like the mock because of the padding around the wizard, and also that it creates less real estate to view the wizard contents (once that comes in)
<img width="1552" alt="Screenshot 2023-06-22 at 10 36 54 AM" src="https://github.com/stackrox/stackrox/assets/4805485/c6d32d9a-b21f-493e-a7fa-7cb44dbaef01">

Reply:
Ah, this seems like a case where the PageSection devs and the Wizard devs on the PatternFly team had differing assumptions about who should handle margins and padding.